src/tools/rustfmt/src/expr.rs

```diff
@@ -399,6 +399,7 @@ pub(crate) fn format_expr(
             }
         }
         ast::ExprKind::Underscore => Some("_".to_owned()),
+        ast::ExprKind::IncludedBytes(..) => None,
```

Comment:
```suggestion
        ast::ExprKind::IncludedBytes(..) => unreachable!(),
```
I suspect that rustfmt doesn't run expansion and can never see this variant.